mobile.html

```diff
@@ -0,0 +1,181 @@
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="underline">
+                <div class="line"></div>
+            </div>
+        </div>
+        <div class="skills">
+            <div class="lang-skills">
+                <div class="lang">
+                    <p>Language</p>
+                </div>
+                <div class="language">
+                    <div class="lang-cont">
+                        <div class="code-lang">
+                            <p>Javascript</p>
+                        </div>
+                        <div class="code-lang2">
+                            <p>Ruby</p>
+                        </div>
+                        <div class="code-lang3">
+                            <p>HTML</p>
+                        </div>
+                        <div class="code-lang4">
+                            <p>CSS</p>
+                        </div>
+
+                    </div>
+
+                </div>
+                <div class="frameworks">
+                    <div class="frame-head">
+                        <div class="frame-lang">
+                            <p>Frameworks</p>
+                        </div>
+                    </div>
+                    <div class="frame-cont">
+                        <div class="frame-lang1">
+                            <p>Bootstrap</p>
+                        </div>
+                        <div class="frame-lang2">
+                            <p>Ruby on Rails</p>
+                        </div>
+                        <div class="frame-lang3">
+                            <p>Rspec</p>
+                        </div>
+                        <div class="frame-lang4">
+                            <p>CapyBara</p>
+                        </div>
+                        <div class="frame-lang5">
+                            <p>Selenuim</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="my-skills">
+                    <div class="skill-name">
+                        <p>Skills</p>
+                    </div>
+                    <div class="skill-cont">
+                        <div class="skill1">
+                            <p>Codekit</p>
+                        </div>
+                        <div class="skill2">
+                            <p>GitHub</p>
+                        </div>
+                        <div class="skill3">
+                            <p>Codepen</p>
+                        </div>
+                        <div class="skill4">
+                            <p>GitLab</p>
+                        </div>
+                        <div class="skill5">
+                            <p>Terminal</p>
+                        </div>
+                    </div>
```

Comment:
Instead of using only `div` for your lists, you should use `ul` list with `li` elements to all your lists of projects technologies, social media icons, and so on. For instance:
```
<ul>
  <li>...Facebook...</li>
  <li>...Github...</li>
   ... social media links missing ....
</ul>
```
```
<ul>
  <li>...Bootstrap...</li>
  <li>...Github...</li>
   ... stack links/icons missing ....
</ul>
```